doc-examples/hibernate-reactive-example-java/build.gradle

```diff
@@ -22,8 +22,14 @@ dependencies {
     implementation "io.micronaut.reactor:micronaut-reactor"
     compileOnly mnSerde.micronaut.serde.api
     runtimeOnly "ch.qos.logback:logback-classic"
-    runtimeOnly libs.drivers.vertx.pg
+    runtimeOnly libs.drivers.vertx.mysql
 
-    testImplementation platform(libs.testcontainers.bom)
-    testImplementation libs.testcontainers.postgresql
+    // Testcontainers wait function needs the driver
+    runtimeOnly libs.drivers.jdbc.mysql
+}
+
+micronaut {
+    testResources {
+        additionalModules.add(HIBERNATE_REACTIVE_MYSQL)
```

Comment:
Can this be detected automatically?

Reply:
It doesn't work without explicitly setting additionalModules.
Without additionalModules:
`[36m17:05:35.442[0;39m [1;30m[ForkJoinPool.commonPool-worker-7][0;39m [34mINFO [0;39m [35mi.m.t.e.TestResourcesResolverLoader[0;39m - Loaded 1 test resources resolvers: io.micronaut.testresources.testcontainers.GenericTestContainerProvider`
With additionalModules:
`[36m17:07:26.148[0;39m [1;30m[ForkJoinPool.commonPool-worker-8][0;39m [34mINFO [0;39m [35mi.m.t.e.TestResourcesResolverLoader[0;39m - Loaded 3 test resources resolvers: io.micronaut.testresources.hibernate.reactive.mysql.HibernateReactiveMySQLTestResourceProvider, io.micronaut.testresources.mysql.MySQLTestResourceProvider, io.micronaut.testresources.testcontainers.GenericTestContainerProvider`